bin/functions/load-config.py

```diff
@@ -327,152 +307,231 @@ def generate_optional_value():  # get some critical values from environment or m
-                    HibenchConfRef["hibench.hadoop.examples.test.jar"]= "Inferred by: " + HibenchConf['hibench.hadoop.home']+"/share/hadoop/mapreduce2/hadoop-mapreduce-client-jobclient*-tests.jar"
-                elif HibenchConf["hibench.hadoop.version"] == "hadoop1":
-                    HibenchConf["hibench.hadoop.examples.test.jar"] = OneAndOnlyOneFile(HibenchConf['hibench.hadoop.home'] + "/share/hadoop/mapreduce1/hadoop-examples-*.jar")
-                    HibenchConfRef["hibench.hadoop.examples.test.jar"]= "Inferred by: " + HibenchConf['hibench.hadoop.home']+"/share/hadoop/mapreduce1/hadoop-mapreduce-client-jobclient*-tests.jar"
-            elif HibenchConf['hibench.hadoop.release'].startswith('hdp'): # HDP release
-                HibenchConf["hibench.hadoop.examples.test.jar"] = OneAndOnlyOneFile(HibenchConf['hibench.hadoop.home'] + "/hadoop-mapreduce-client-jobclient-tests.jar")
-                HibenchConfRef["hibench.hadoop.examples.test.jar"]= "Inferred by: " + HibenchConf['hibench.hadoop.home']+"/hadoop-mapreduce-client-jobclient-tests.jar"
-
+        if HibenchConf['hibench.hadoop.release'] == 'apache':
+            HibenchConf["hibench.hadoop.examples.test.jar"] = OneAndOnlyOneFile(HibenchConf[
+                                                                                        'hibench.hadoop.home'] + "/share/hadoop/mapreduce/hadoop-mapreduce-client-jobclient*-tests.jar")
+            HibenchConfRef["hibench.hadoop.examples.test.jar"] = "Inferred by: " + HibenchConf[
+               'hibench.hadoop.home'] + "/share/hadoop/mapreduce/hadoop-mapreduce-client-jobclient*-tests.jar"
+        # CDH release
+        elif HibenchConf['hibench.hadoop.release'].startswith('cdh'):
+            HibenchConf["hibench.hadoop.examples.test.jar"] = OneAndOnlyOneFile(HibenchConf[
+                                                                                        'hibench.hadoop.home'] + "/share/hadoop/mapreduce2/hadoop-mapreduce-client-jobclient*-tests.jar")
+            HibenchConfRef["hibench.hadoop.examples.test.jar"] = "Inferred by: " + HibenchConf[
+                        'hibench.hadoop.home'] + "/share/hadoop/mapreduce2/hadoop-mapreduce-client-jobclient*-tests.jar"
+        # HDP release
+        elif HibenchConf['hibench.hadoop.release'].startswith('hdp'):
+            HibenchConf["hibench.hadoop.examples.test.jar"] = OneAndOnlyOneFile(
+                    HibenchConf['hibench.hadoop.home'] + "/hadoop-mapreduce-client-jobclient-tests.jar")
+            HibenchConfRef["hibench.hadoop.examples.test.jar"] = "Inferred by: " + HibenchConf[
+                    'hibench.hadoop.home'] + "/hadoop-mapreduce-client-jobclient-tests.jar"
+
+def probe_sleep_job_jar():
     # set hibench.sleep.job.jar
     if not HibenchConf.get('hibench.sleep.job.jar', ''):
-        if HibenchConf['hibench.hadoop.release'] == 'apache' and HibenchConf["hibench.hadoop.version"] == "hadoop1":
+        if HibenchConf['hibench.hadoop.release'] == 'apache':
```

Comment:
According to the original condition, this only applies to hadoop1. For hadoop2, the path is different. We should remove this.